inscriptions/ISic030013.xml

```diff
@@ -109,9 +111,9 @@
                                 <placeName type="modern">Selinunte</placeName>
                                 <geo>37.5853775, 12.8298666</geo>
                             </origPlace>
-                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0600" notAfter-custom="-0400" precision="low">6th—5th century BCE</origDate> <!-- check date -->
+                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0450" notAfter-custom="-0450">450 BCE</origDate> <!-- from Arena-->
```

Comment:
dating cannot actually be done ad annum, it is approximate. In such cases, we suggest entering a range according to what the editor says. In this case: 475-450